utils/src/main/scala/za/co/absa/enceladus/utils/error/UDFLibrary.scala

```diff
@@ -18,34 +18,36 @@ package za.co.absa.enceladus.utils.error
 import org.apache.spark.sql.api.java._
 import org.apache.spark.sql.types._
 import org.apache.spark.sql.{Row, SparkSession}
-
 import scala.collection.mutable
 
+
 case class UDFLibrary()(implicit val spark: SparkSession) {
 
-  spark.udf.register("stdCastErr", { (errCol: String, rawValue: String) =>
+  spark.udf.register(UDFunctionNames.stdCastErr, { (errCol: String, rawValue: String) =>
```

Comment:
Two questions about the UDFunctionNames:
- Why is it `UDFunctionNames` and not `UDFNames`? Since the full abbreviation is UDF
- Why not use `import UDFunctionNames._` and omit the object name in calling of `stdCastErr` and others?